assignment/assignment1/client/src/scenes/loginPage/Form.tsx

```diff
@@ -73,29 +79,35 @@ const Form = () => {
       //   }
       // }
     }
-    const username = formData.get("username");
-    const password = formData.get("password");
-    // formData.append("picturePath", values.picture!.name);
-    const savedUserRespone = await fetch(
-      `http://localhost:${PORT}/auth/register`,
-      {
-        method: "POST",
-        // body: formData,
-        body: JSON.stringify({
-          username,
-          password,
-        }),
-        headers: {
-          "Content-Type": "application/json",
-        },
+    try {
+      const username = formData.get("username") == null ? '' : formData.get("username").toString();
+      const password = formData.get("password") == null ? '' : formData.get("username").toString();
```

Comment:
just a small suggestion, can use this instead to assign values
`const username = formData.get("username").toString() ?? ''`